src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -412,7 +412,20 @@ $ npm install @sentry/browser
 $ npm install @sentry/apm
 ```
 
-#### Sending Traces/Transactions/Spans
+OR
+
+Alternativly if you can't use npm packages you can use our pre-built CDN bundle that combines both `@sentry/browser` and `@sentry/apm`:
+
+```html
+<script src="https://browser.sentry-cdn.com/{% sdk_version sentry.javascript.browser %}/bundle.apm.min.js" crossorigin="anonymous"></script>
+```
+
+*NOTE:* The `Tracing` integration is available then under `Sentry.Integrations.Tracing`.
+
+#### Automatic Instrumentation
+
+For Javascript Browser we provide you with an integration that does automatic instrumentation creating `pageload` and `navigation` transaction containing Spans for XHR/fetch, Performance API (marks, measures, FCP, LCP) and resources timings (also provided by the Performance API). The Tracing
+intergration is specific to Browser so it doesn't work on Node.js.
```

Comment:
```suggestion
For `@sentry/browser`, we provide an integration called `Tracing` that does
automatic instrumentation creating `pageload` and `navigation` transactions
containing spans for XHR/fetch requests and Performance API entries such as
marks, measures and resource timings.

The `Tracing` integration is specific to `@sentry/browser` and does not work
with `@sentry/node`.
```

- Talk in terms of SDK package names for consistency;
- Introduce the name of the integration, `Tracing`, before saying that it is specific to `@sentry/browser` and before saying where it resides (later down below);
- Reword part about Performance API to avoid excess of parentheses and more generically talk about entries;
- Move note about `Tracing` is only for browser into a new paragraph for visibility. Could consider a NOTE box, though it might be too much visual noise in this case;
- Typo fixes;